doc/user_guide/customization.rst

```diff
@@ -290,3 +290,88 @@ You can remove the legend entirely by submitting a null value.
       y='petalLength',
       color=alt.Color('species', legend=None),
   )
+
+Adjusting the width of Bar Marks
+--------------------------------
+The width of the bars in a bar plot are controlled through the `size` property in the `mark_bar()`:
```

Comment:
Please change to double back-ticks for literals like ``size``.

For referencing methods, please use
```
:meth:`~Chart.mark_bar`
```
which will automatically be turned into a clickable a cross-reference.